README.md

```diff
@@ -1,14 +1,36 @@
 
 # jpo-security-svcs
-This module expopsed a RESTful API for performing cryptographic functions. The following paths identify the functions:
+This module exposes a RESTful API for performing cryptographic functions. The cryptographic functions may be carried out against an on-prem instance, or against a remote instance. Note that for the remote signing use case, MTLS will be used to secure the communications. This requires some additional work on the part of configuration. The following paths identify the functions:
 |Verb|path|Content Type|Functionality|Request Body Format|Response Body Format|
 |--|--|--|--|--|--|
 |POST|/sign|application/json|signs data provided in the body of the request|{"message":"Base64 encoded unsigned data"}|{"result": "Base64 Encoded Signed Data"}
 
+## Mutual TLS Authentication
+For enhanced security, MTLS is used to communicate with a remote signing system. This requires properly configured certificates from the remote system to property perform - work with your remote signing authority to obtain necessary certificates. 
+
+### Certificate Conversion
+Certificates can be in several different formats, including the widely used PEM format. Because this is a Java application, the signing certificates must be in the Java Keystore format to be used. The following commands using [OpenSSL](https://www.openssl.org/) and the Java keytool will convert a PEM file to a Java Keystore file.
+
+First we convert the PEM file to a PKCS12 file:
```

Comment:
Making this a list of steps rather than using "first" and "then" would make it more readable.

Example:

1. Convert the PEM file to a PKCS12 file:
```
openssl pkcs12 -export -in <cert.pem> -inkey <key.pem> -out <certificate.p12> -name <alias>
```
2. Convert the PKCS12 file to a Java Keystore file:
```
keytool -importkeystore -srckeystore <certificate.p12> -srcstoretype pkcs12 -destkeystore <cert.jks>
```

Reply:
Updated